modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -166,12 +166,15 @@ def expand_va_url(url)
     def notify_slack(old_form_url, new_form_url, form_name)
       return unless Settings.va_forms.slack.enabled
 
-      slack_details = {
-        class: self.class.name,
-        alert: "#{form_name} has changed from #{old_form_url} to #{new_form_url}"
-      }
-
-      VAForms::Slack::Messenger.new(slack_details).notify!
+      begin
+        slack_details = {
+          class: self.class.name,
+          alert: "#{form_name} has changed from #{old_form_url} to #{new_form_url}"
+        }
+        VAForms::Slack::Messenger.new(slack_details).notify!
+      rescue => e
+        Rails.logger.error("#{self.class.name} failed to notify Slack for form update", e)
+      end
```

Comment:
Not including specific error handling for this was an oversight in my original PR that updated the Slack alert.

Reply:
Before this change, an error here should have still fallen into the `rescue` block of the method that had called it, but specific handling here will allow an individual form record to still be successfully updated if the Slack notification fails.